src/test/mod.rs

```diff
@@ -43,7 +43,11 @@ pub(crate) fn assert_redirect(path: &str, expected_target: &str, web: &TestFront
     let status = response.status();
 
     // Reqwest follows redirects automatically
```

Comment:
This might be a bit nit-picky... but I find this comment confusing now. This comment originally had a slightly different place/context. Now it seems to tell us something about the conversion of `response` to `redirect_target`, but the redirect has already happened by the time we get here.

I would propose:
- move this comment up a bit, maybe at `let response = web.get(path).send()?;`?
- perhaps add a (short) comment at `let redirect_target = if ...` to clarify why we sometimes take the full URL and sometimes the path only.

Reply:
I pushed a change that comments this, does that make it more clear?

Reply:
Yeah, this is great :+1: